apps/web/src/pages/templates/components/phone-simulator/PhonePlatformSwitch.tsx

```diff
@@ -0,0 +1,99 @@
+import { ComponentProps, forwardRef } from 'react';
+import styled from '@emotion/styled';
+import { colors, shadows } from '@novu/design-system';
+
+import { AndroidIcon } from './AndroidIcon';
+import { AppleIcon } from './AppleIcon';
+
+export type IPhonePlatformSwitchProps = Pick<ComponentProps<'input'>, 'checked' | 'onChange' | 'className' | 'style'>;
+
+const switchContainerPadding = 5;
+const thumbWidth = 48;
+const thumbHeight = 28;
+
+const AppleIconStyled = styled(AppleIcon)`
+  min-width: 16px;
+  width: 16px;
+  height: 16px;
+`;
+
+const AndroidIconStyled = styled(AndroidIcon)`
+  min-width: 16px;
+  width: 16px;
+  height: 16px;
+`;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Perhaps each icon can just expose a `size` prop that gets piped into its styles?‏